README.md

```diff
@@ -41,6 +41,46 @@ Interact with people within a 100-meter radius! Get real-time locations of nearb
 
 Time to mingle, chat, and rate! Have fun exploring the world around you. 🌍🎉
 
+## Troubleshooting 🤔
+
+If you encounter any issues while using the app, this section will help you find quick solutions to common problems.
+
+### 1. App Crashes on Device Configuration Changes
+**Issue:** The app crashes when the device orientation or configuration changes.  
+**Solution:**  
+- Ensure that `android:configChanges="orientation|screenSize"` is added in the AndroidManifest.xml for the affected activities.
+- Consider using ViewModel or handling configuration changes appropriately in the activity lifecycle.
+
+### 2. User's Location is Not Detected
+**Issue:** The app is not able to detect the user's location.  
+**Solution:**  
+- Make sure that location permissions are granted in the device settings.
+- If the location is off, prompt the user to enable location services.
+
+### 3. Profile Pictures Still Visible Despite Disabling
```

Comment:
This is an issue which will be resolved soon. Putting it in this section doesn't make any sense ig.